### ISSUE TITLE
train nima on tid2013 ?

### ISSUE THREAD

**Issue by ranjiewwen** [hcl14/AVA-and-TID2013-image-quality-assessment#1]
- fisrt thanks you code ! but when i read you nima_tid2013_regression.py , i am confused.
- where is the `tid2013_1000.pickle` and `train_test_split_cs.pickle` ?

**Comment by hcl14** [comment]
@ranjiewwen Hi, I'm sorry for the very dirty and disorganized code I uploaded here. Just was too tired to organize it well when finished. I've added both `tid2013_1000.pickle` generation script and file itself. It uses `mos_with_names.txt` and `mos_std.txt` which are present in original tid dataset. `train_test_split_cs.pickle` was used to continue training and was generated by training script. After that, I was commenting/uncommenting parts used for loading labels and saved checkpoints and continued training. The name `tid2013_1000` suggests that I tried to do histogram prediction for that strange variances in TID dataset, but that did not yield better results than just regression. 

Please excuse me again for these awful scripts, may be I will organize them one day. If you have any questions/problems with training, please ask me here. I will try to remember and fix that.

**Comment by ranjiewwen** [comment]
@hcl14 thanks for your reply !  i will try your code again. 
- by the way. recently i am also reimplement [nima there](https://github.com/ranjiewwen/TF_NIMA), i use tensorflow native api and use vgg model on tid2013. but when i train can get relative high srocc on one batch data . but when evaluate on 20% datasets tid2013 get very low result srocc. i don't know why ,maybe make some mistakes or overfit . but i have check many times , if you are interesting , can you give me some advices ? thanks again!
